quipucords/scanner/discovery.py

```diff
@@ -117,6 +117,14 @@ def discovery(self):
             cred = hc_serializer.data
             connected, remaining = connect(remaining, cred, connection_port,
                                            forks=forks)
+
```

Comment:
This also runs for a host scan. So need to toggle based on that.

Reply:
Put check for discovery scan